plugins/dualGovernance/hooks/useCanCreateProposal.tsx

```diff
@@ -1,19 +1,24 @@
 import { Address, keccak256, toHex } from "viem";
 import { useState, useEffect } from "react";
-import { useBalance, useAccount, useReadContracts, useConfig } from "wagmi";
-import { readContract } from "@wagmi/core";
+import {
+  useBalance,
+  useAccount,
+  useReadContracts,
+  useReadContract,
+} from "wagmi";
 import { OptimisticTokenVotingPluginAbi } from "@/plugins/dualGovernance/artifacts/OptimisticTokenVotingPlugin.sol";
 import { DaoAbi } from "@/artifacts/DAO.sol";
 import { PUB_CHAIN, PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS } from "@/constants";
 
 export function useCanCreateProposal() {
-  const config = useConfig();
   const { address } = useAccount();
+  if (!address) return false;
```

Comment:
This is something I wanted to talk about and I forgot. I noticed that we are not using the types from the ABIs because they need to be exported as plain const to work. When I activated the ABI types I found a lot of errors like this one, sending an incorrect type (unknown in this case) to useBalance, etc. I'm going to create a ticket to fix all of them. 

I didn't know about `query: {enabled: condition}`. Definitely is the right solution here. I will change it.

Reply:
I created this ticket for the ABI types: https://linear.app/aragon/issue/RD-211/abi-types-not-working
And this one to check the Wagmi hooks: https://linear.app/aragon/issue/RD-212/check-we-are-using-wagmi-hooks-properly